src/pages/users/signUp.tsx

```diff
@@ -9,23 +9,10 @@ import { putUserInfo } from "@/api/clientRequests/postPutUserInfo";
 import FormHeader from "./formHeader";
 import { setUserNameAction } from "@/redux/store/reducers/userReducer";
 
-const useFields: () => [
-  string,
-  React.Dispatch<React.SetStateAction<string>>,
-  string,
-  React.Dispatch<React.SetStateAction<string>>,
-  string,
-  React.Dispatch<React.SetStateAction<string>>
-] = () => {
+const SignUp = (props: ISignFormProps) => {
   const [login, setLogin] = useState("");
   const [password, setPassword] = useState("");
   const [password2, setPassword2] = useState("");
```

Comment:
same with naming

Reply:
replaced with password and repeat password